read-historical-data.md

```diff
@@ -152,3 +152,17 @@ Pay special attention to the following:
     > **Note:**
     >
     > You should use `@@` instead of `@` before `tidb_snapshot` because `@@` is used to denote the system variable while `@` is used to denote the user variable.
+
+## Restoring old versions
+
+The first thing you probably want to do when you need to restore from an older version is to make sure GC doesn't remove it while you are working on it. This can be done by setting the GC as is done in the example below. Don't forget to set it back to the previous value after the restore.
+
+```sql
+SET GLOBAL tidb_gc_life_time="60m";
+```
+
+For simple cases you could use `SELECT` after setting the `tidb_snapshot` variable and copy-paste the output or use `SELECT ... INTO LOCAL OUTFLE` and use `LOAD DATA` to import the data later on.
+
+Another option is to use [dumpling to export a historical snapshot](/dumpling-overview.md#export-historical-data-snapshot-of-tidb). This is a good tool for exporting larger sets of data.
+
+If you plan to use this feature to do restores you probably want to increase the GC life time from the default 10m to something like half an hour.
```

Comment:
Please state the risks when doing this. For example by adding a sentence:

"This will result in additional versions of rows being retained, which may increase disk space requirements slightly. The performance of certain options such as scans may also degrade, when additional versions of the same rows need to be skipped when reading".